osfoffline/views/rsc/gui/startscreen_gui/startscreen.ui

```diff
@@ -41,45 +41,50 @@
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
-      <x>10</x>
-      <y>20</y>
-      <width>141</width>
+      <x>30</x>
+      <y>50</y>
+      <width>110</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
-     <string>Personal Access Token</string>
+     <string>Username (email):</string>
```

Comment:
While you're in QT designer, could you also change the title of the dialog to something other than "startscreen"?

(Technically another ticket, but this will save time later) 

<img width="654" alt="screen shot 2015-11-11 at 12 30 18 am" src="https://cloud.githubusercontent.com/assets/2957073/11084600/6f34c076-880b-11e5-8d57-195a03349672.png">